test/lint/test_runner/src/main.rs

```diff
@@ -180,6 +185,31 @@ fn lint_subtree() -> LintResult {
     }
 }
 
+fn lint_py_mut_arg_default() -> LintResult {
+    let bin_name = "ruff";
+    let checks = ["B006", "B008"]
+        .iter()
+        .map(|c| format!("--select={}", c))
+        .collect::<Vec<_>>();
+    let files = check_output(git().args(["ls-files", "--", "*.py"]))?;
+
+    let mut cmd = Command::new(bin_name);
+    cmd.arg("check").args(checks).args(files.lines());
```

Comment:
nit:

```suggestion
    cmd.arg("check").args(checks).args(files.lines()).arg("--quiet");
```

This would silence unhelpful messages about fixes not being available.